### PR TITLE
Adding helm trigger documentation

### PR DESCRIPTION
**Pull request by anais-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-triggers/docs/

File: _docs/configure-ci-cd-pipeline/triggers/helm-triggers.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: "Helm Trigger"
+description: ""
+group: configure-ci-cd-pipeline
+sub_group: triggers
+toc: true
+---
+
+## Manage Helm Triggers with Codefresh UI
+
+It is possible to define and manage Helm pipeline triggers with the Codefresh UI.
```
Comment (kostis-codefresh) [inline _docs/configure-ci-cd-pipeline/triggers/helm-triggers.md:11]: Add a short paragraph here on what is the end result/value for developers. That you can create pipelines that respond to Helm events. For example you can have one pipeline that creates a docker image and chart, and as soon as the chart is created another pipeline takes over for the actual deployment.

Comment (kostis-codefresh) [reply _docs/configure-ci-cd-pipeline/triggers/helm-triggers.md:11]: Also add a short notice that at the moment, only artifactory Helm charts are supported.

File: _data/nav.yml [hunk 2]
```diff
@@ -28,6 +28,8 @@
           url: "/azure-triggers"
         - title: "Quay Triggers"
           url: "/quay-triggers"
+        - title: "Helm Triggers"
+          url: "/helm-triggers"
```
Comment (kostis-codefresh) [inline _data/nav.yml:32]: You should also add a link in triggers.md

File: _docs/configure-ci-cd-pipeline/triggers/helm-triggers.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+url="/images/pipeline/triggers/jfrog/view-trigger-dialog.png"
+alt="Codefresh webhook URL"
+max-width="50%"
+%}
+
+Now we must set JFrog Artifactory to call this URL when an event takes place. This can either be done through the [JFrog Artifactory webhook plugin](https://codefresh.io/docs/docs/configure-ci-cd-pipeline/triggers/jfrog-triggers/) or through [setting up Webhooks](https://www.jfrog.com/confluence/display/JFROG/Webhooks) in the UI.
+
+### Triggering a Codefresh pipeline with an Artifactory push
+
+Now, every time you push a Helm chart to the selected Artifactory repository, manually, with Codefresh or any other CI/CD tool, Codefresh will trigger execution of all pipelines associated with that Artifactory Push trigger event.
```
Comment (kostis-codefresh) [inline _docs/configure-ci-cd-pipeline/triggers/helm-triggers.md:54]: Also add a "what to read next" section here.

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-triggers/docs/

**Pull request merged**